lte/gateway/c/oai/lib/secu/nas_stream_eia1.c

```diff
@@ -117,12 +118,13 @@ int nas_stream_encrypt_eia1(
   uint64_t c;
   uint64_t M_D_2;
   int rem_bits;
-  uint32_t mask = 0;
-  uint32_t* message;
+  uint32_t mask     = 0;
+  uint32_t* message = (uint32_t*) malloc(
+      sizeof(uint32_t) * ((stream_cipher->blength >> 5) + 1));
+
+  /* copy message to avoid memory alignment issues */
+  memcpy(message, stream_cipher->message, ceil(stream_cipher->blength / 8.0));
```

Comment:
Again, wondering if alignment issue here was expected or not?

Reply:
Same here, input is a stream of bytes that guarantees no alignment